src/fake-trips-helper.js

```diff
@@ -30,6 +24,20 @@ const validateTrip = (start, end) => {
   if (distance < 1600) {
     return false;
   } else {
-    return {distance};
+    return Math.round(distance);
+  }
+}
+
+module.exports = class FakeTrip {
+  constructor () {
+    let pickup = generateCoordinatesWithPeakZone();
+    let dropoff = generateCoordinatesWithPeakZone();
```

Comment:
At peak time there is often a net inflow (morning) or outflow (evening) to a high density work area.  At the moment the model simulates a high likelihood of rides both originating and terminating within the peak zone at peak hours.  This is possible and fine for now.  Later perhaps, we could run another simulation with net inflows or outflows.